public/style.css

```diff
@@ -31,11 +43,132 @@ ul li{
   width: 350px;
   cursor: pointer;
   box-shadow: 0 0 5px rgb(133, 8, 8);
-  
 }
 
-ul li:hover{
- text-decoration: underline #d63147;
- background-color: transparent;
- transform: scale(1.1);
+ul li:hover {
+  text-decoration: underline #d63147;
+  background-color: transparent;
+  transform: scale(1.1);
+}
+
+#player-name-input,
+#confirm-button,
+#start-quiz-button {
+  margin-bottom: 10px;
+  padding: 15px;
+  font-size: 16px;
+  border-radius: 10px;
+  border: none;
+}
+#player-name-input {
+  border: 3px dashed rgb(250, 208, 44);
+  padding: 8px 10px;
+  font-size: 16px;
+}
+
+#player-name-input:focus {
+  border: 3px dashed rgb(250, 208, 44);
+  outline: none;
+}
+button {
+  background-color: rgb(128, 204, 66);
+  cursor: pointer;
+  padding: 10px 20px;
+  font-size: 16px;
+  border-radius: 10px;
+  width: 50%;
+  border: none;
+  cursor: not-allowed;
+}
+#confirm-button {
+  background-color: rgb(194, 193, 193);
+  cursor: pointer;
+  padding: 10px 20px;
+  font-size: 16px;
+  border-radius: 10px;
+  width: 50%;
+  border: none;
+  cursor: not-allowed;
```

Comment:
- We don't want to add `cursor: not-allowed;` to all buttons. We want to add it to only disabled buttons. So please delete this line.